tools/generate-helpers/pg-table-bindings/main.go

```diff
@@ -94,6 +102,15 @@ type properties struct {
 	// Indicates that we should just generate the singleton store
 	SingletonStore bool
 
+	// Migration root
+	MigrateRoot string
+
+	// Where the data are migrated from in the format of "database:bucket", eg, \"rocksdb\", \"dackbox\" or \"boltdb\"")
+	MigrateFrom string
+
+	// The unique sequence number to migrate all tables to Postgres
+	MigrateSeq int
```

Comment:
How does this manifest when upgrading from one version to another on postgres?

For example, if I were to update a table schema, do we have to update this seq-num? If yes, only on a concerned table or all tables?

Reply:
So to clarify, this is only for the initial case of migrating Rocks to Postgres.  Correct?  As such I think I find the n_01_to_n_02_postgres_clusters naming to be confusing as we aren't really going from postgres table version 1 to postgres table version 2, but we are simply working on the first table in the postgres schema hierarchy which is the clusters table.    But I will admit I don't have a great feel for how migrations work now.

Reply:
Thanks for the clarification @c-du. I really wanted to understand the post-release process. For example, if I were to add 3 new independent tables, should each table have a unique seq number or can I push all those tables through one sequence number?

Reply:
If you add new tables after release, you do not need to add new sequence numbers, that is, you do not need to add migration. Once you added the schema, the migrator will pick up and apply the schema before the central starts.